Makefile

```diff
@@ -91,3 +90,22 @@ db-compress: assets/$(DB_TYPE)/$(DB_TYPE).db assets/$(DB_TYPE)/metadata.json
 .PHONY: db-clean
 db-clean:
 	rm -rf cache assets
+
+.PHONY: db-fetch-vuln-list-master
+db-fetch-vuln-list-master:
+	mkdir -p cache/vuln-list
+	wget -qO - https://github.com/aquasecurity/vuln-list/archive/master.tar.gz | tar xz -C cache/vuln-list --strip-components=1
+
+.PHONY: db-fetch-vuln-list-fixed
+db-fetch-vuln-list-fixed:
+	mkdir -p cache/vuln-list
+	wget -qO - https://github.com/aquasecurity/vuln-list/archive/8b786696ee5c47495e19f9453b467e729b844a17.tar.gz | tar xz -C cache/vuln-list --strip-components=1
```

Comment:
I didn't know we can download `tar.gz` specifying a commit hash. Nice!

Reply:
Yes I also recently found this out!